src/styles/_catalog.scss

```diff
@@ -1,22 +1,25 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  grid-template-columns: repeat(auto-fill, 200px);
 
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
+    display: grid;
+    place-items: center;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
```

Comment:
You don't have a margin here because now you have a gap.
![image](https://user-images.githubusercontent.com/99403198/203974080-eba08ec3-d738-450d-be33-271034980b88.png)